lsp-ui-doc.el

```diff
@@ -302,7 +302,10 @@ Because some variables are buffer local.")
        (if (and language (not (string= "text" language)))
            (format "```%s\n%s\n```" language string)
          string))
-      (t (lsp--render-string (lsp-ui-doc--inline-formatted-string string) language))))))
+      ;; For other programming languages
+      (language (lsp--render-string (lsp-ui-doc--inline-formatted-string string) language))
+      ;; For default markdown content
+      (t (lsp--render-element (lsp-ui-doc--inline-formatted-string string)))))))
```

Comment:
Language is `nil` when the string content is in markdown. Render it with function `lsp--render-element` and use function `lsp--render-string` for the rest.